priv/posts/marketing/20210714233950_getting-to-know-paulo-oliveira.md

```diff
@@ -11,7 +11,7 @@ Getting to know to Paulo F. Oliveira
 ---
 
 
-<img src="/images/getting-to-know/poliveira.png" class="img-fluid" alt="Paulo F. Oliveira"/>
+<img src="/images/getting-to-know/poliveira.png" width="100%" height="auto" alt="Paulo F. Oliveira"/>
```

Comment:
Why these changes?

Reply:
The Markdown sanitizer removes the class="img-fluid"  from the img tag, so the images cut out, width="100%" height="auto" fixes it.